airflow-core/patch.py

```diff
@@ -0,0 +1,52 @@
+import os
```

Comment:
Previous comment in the melange invoking the patch - but would need some info here on the source of this patch / info where it was derived, and all the various settings it's adding in addition to head - so sustaining know what to do with this package

Reply:
Removed the patch as after multiple checks with upstream, airflow-core doesn't need it, if at all, as suggested by upstream project too, the user will have to add the providers on their own